### PR TITLE
Add interventions and proposals count to meetings xls

### PR DESCRIPTION
**Pull request by beagleknight**
# What and why

Added missing fields to the meetings xls export:
- Interventions count
- Proposals count
# GIF Tax

### CODE REVIEW

File: app/controllers/moderation/meetings_controller.rb [hunk 1]
```diff
@@ -38,7 +40,9 @@ def index
               row.push meeting.title
               row.push meeting.address
               row.push meeting.attendee_count
+              row.push meeting.interventions
               row.push meeting.organization_count
+              row.push meeting.proposals.count
```
Comment (josepjaume) [inline app/controllers/moderation/meetings_controller.rb]: I don't think this is gonna scale :S. We should probably cache this or something...

Comment (beagleknight) [inline app/controllers/moderation/meetings_controller.rb]: @josepjaume I added a `counter_cache` but I discovered a nice rails bug: https://github.com/rails/rails/issues/7618 It wasn't solved on Rails 3.1...neither in 4.2... maybe in 5?

File: app/models/meeting.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Meeting < ActiveRecord::Base
 
   has_many :meeting_proposals, dependent: :destroy
   accepts_nested_attributes_for :meeting_proposals
-  has_many :proposals, through: :meeting_proposals
+  has_many :proposals, through: :meeting_proposals, before_remove: proc { |m, p| MeetingProposal.where(meeting: m, proposal: p).destroy_all }
```
Comment (josepjaume) [inline app/models/meeting.rb:12]: Depressing

**Comment by josepjaume** [comment]
👍

**Pull request merged**